hermes-tracker-elasticsearch/src/main/java/pl/allegro/tech/hermes/tracker/elasticsearch/consumers/ElasticsearchLogRepository.java

```diff
@@ -9,28 +9,31 @@
 import pl.allegro.tech.hermes.tracker.consumers.LogRepository;
 import pl.allegro.tech.hermes.tracker.consumers.MessageMetadata;
 import pl.allegro.tech.hermes.tracker.elasticsearch.ElasticsearchQueueCommitter;
+import pl.allegro.tech.hermes.tracker.elasticsearch.IndexFactory;
 import pl.allegro.tech.hermes.tracker.elasticsearch.LogSchemaAware;
+import pl.allegro.tech.hermes.tracker.elasticsearch.SchemaManager;
 import pl.allegro.tech.hermes.tracker.elasticsearch.metrics.Gauges;
 import pl.allegro.tech.hermes.tracker.elasticsearch.metrics.Timers;
 
 import java.io.IOException;
 
 import static org.elasticsearch.common.xcontent.XContentFactory.jsonBuilder;
-import static pl.allegro.tech.hermes.api.SentMessageTraceStatus.*;
+import static pl.allegro.tech.hermes.api.SentMessageTraceStatus.DISCARDED;
+import static pl.allegro.tech.hermes.api.SentMessageTraceStatus.FAILED;
+import static pl.allegro.tech.hermes.api.SentMessageTraceStatus.INFLIGHT;
+import static pl.allegro.tech.hermes.api.SentMessageTraceStatus.SUCCESS;
 import static pl.allegro.tech.hermes.tracker.elasticsearch.DocumentBuilder.build;
-import static pl.allegro.tech.hermes.tracker.elasticsearch.LogSchemaAware.TypedIndex.SENT_MESSAGES;
 
 public class ElasticsearchLogRepository extends BatchingLogRepository<XContentBuilder> implements LogRepository, LogSchemaAware {
```

Comment:
consider adding Sent to have more descriptive name for this and another ElasticseachLogRepository